torch/multiprocessing/reductions.py

```diff
@@ -99,7 +99,7 @@ def rebuild_tensor(cls, storage, metadata):
 
 
 def rebuild_cuda_tensor(tensor_cls, tensor_size, tensor_stride, tensor_offset,
-                        storage_cls, storage_device, storage_handle, storage_size_bytes, storage_offset_bytes,
+                        storage_cls, dtype, storage_device, storage_handle, storage_size_bytes, storage_offset_bytes,
```

Comment:
This looks dodgy, for BC/FC.

Reply:
This is only used in ForkingPickler, so it should be ok, right?

Reply:
oops, that's right! you're clear